MooseFS/docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: "3"
+
+services:
+  weba:
+    build:
+      context: ./weba
+    ports:
+      - "8001:5000"
+    volumes:
+      - type: volume
+        source: mfs_data
+        target: /var/lib/mfs
+    depends_on:
+      - chunkserver1
+      - chunkserver2
+    # restart: always
+
+  webb:
+    build:
+      context: ./webb
+    ports:
+      - "8002:5000"
+    volumes:
+      - type: volume
+        source: mfs_data
+        target: /var/lib/mfs
+    depends_on:
+      - chunkserver1
+      - chunkserver2
+    # restart: always
+  mfsmaster:
+    build: ./moosefs-master
+    # image: moosefs/moosefs-master
+    container_name: mfsmaster
+    ports:
+      - "9421:9421"
+      - "9422:9422"
+      - "9419:9419"
+      - "9433:9433"
+    volumes:
+      - type: volume
+        source: mfs_data
+        target: /var/lib/mfs
+    environment:
+      - MFS_ENV=TEST 
+      # In the TEST environment, if metadata.mfs file or other metadata files
+      # like metdata.mfs.back and changelog.0.mfs are missing, the script will
+      # create an EMPTY metadata file!
+      # When MFS_ENV=PROD metdata.mfs file will not be created!
+
+  chunkserver1:
+    build: ./moosefs-chunkserver
+    container_name: chunkserver1
+    volumes:
+      - type: volume
+        source: mfs_data
+        target: /data
+    environment:
+      - LABELS=M
+      #- SIZE=100
+    links:
+      - mfsmaster
+    depends_on:
+      - mfsmaster
+  chunkserver2:
+    build: ./moosefs-chunkserver
+    container_name: chunkserver2
+    volumes:
+      - type: volume
+        source: mfs_data
+        target: /data
+    environment:
+      - LABELS=M,B
+      #- SIZE=100
+    links:
+      - mfsmaster
+    depends_on:
+      - mfsmaster
+
+volumes:
+  mfs_data:
```

Comment:
I think you need to do this with different volumes. We can't have the same volumes mounted to all the containers. This would mean that all the services are deployed on one common machine, with one filesystem. What we instead want is the following
Machine 1: Master server
Machine 2: Chunkserver 1, Web-A
Machine 3: Chunkserver 2, Web-B.

This means we want 3 separate volumes call them mfsdata_master, mfsdata_chunk1, mfsdata_chunk2. These volumes are mounted to the containers as described above.